.github/workflows/jekyll.yml

```diff
@@ -75,8 +75,14 @@ jobs:
 
       - run: pip install linkchecker --user
 
-      - name: Link Checker
-        run: |
-          linkchecker https://duckdb.org/ --config .github/linkchecker/linkchecker.conf --file-output html/utf-8/report.html
-      - run: cat report.html >> $GITHUB_STEP_SUMMARY
+      - name: Link Checker on latest
+        run: linkchecker https://duckdb.org/ --ignore-url 'https://duckdb\.org/docs/archive/.*' --config .github/linkchecker/linkchecker.conf --file-output html/utf-8/report.html
+
+      - run: cat report.html >> ${GITHUB_STEP_SUMMARY}
+        if: success() || failure()
```

Comment:
```suggestion
        if: always()
```

Reply:
Good point, it should still display the summary if the second `linkchecker` step failed.